Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM alpine:3.14
 WORKDIR /app
 ENV GITHUB_ORG='<GITHUB-ORG>'
 ENV GITHUB_REPO='<GITHUB-REPO>'
+ENV GITHUB_CATEGORY='<GITHUB-CATEGORY>'
```

Comment:
maybe use "CATEGORIES" instead of "CATEGORY"?

Reply:
done.